examples/orm-django-quickstart/sample_project/models.py

```diff
@@ -1,16 +1,12 @@
 from django.db import models
-from django_tidb.fields.vector import VectorField
+from django_tidb.fields.vector import VectorField, VectorIndex, CosineDistance, L2Distance
 
 
 class Document(models.Model):
     content = models.TextField()
     embedding = VectorField(dimensions=3)
-
-
-class DocumentWithIndex(models.Model):
-    content = models.TextField()
-    # Note:
-    #   - Using comment to add hnsw index is a temporary solution. In the future it will use `CREATE INDEX` syntax.
-    #   - Currently the HNSW index cannot be changed after the table has been created.
-    #   - Only Django >= 4.2 supports `db_comment`.
-    embedding = VectorField(dimensions=3, db_comment="hnsw(distance=cosine)")
+    class Meta:
+        indexes = [
+            VectorIndex(L2Distance("embedding"), name='idx_l2'),
```

Comment:
Is it possible to omit the name?

Reply:
No, according to the `django.db.models.Index`, the user must set the name for an expression index.
https://github.com/django/django/blob/69bf08e3a32492998871eb91ad84b3c8d8117180/django/db/models/indexes.py#L47-L48